mhs/common/mhs_common/workflow/common.py

```diff
@@ -34,3 +40,31 @@ class CommonWorkflow(abc.ABC):
         :param payload: payload to handle
         """
         pass
+
+    @staticmethod
+    def handle_spine_fault(error: httpclient.HTTPClientError,
+                           logger: IntegrationAdaptorsLogger) -> Tuple[int, AnyStr]:
+        """
+
+        :param error: Tornado's HTTPClient error instance
+        :param logger: Logger to be used to log Spine error(s)
+        :return: Response to external client represented as HTTP status code and body
+        """
+        response_code = error.response.code
+        response_headers = error.response.headers
+        response_body = error.response.body
+        client_message = 'Error(s) returned from Spine. Contact system administrator.'
+
+        if response_code == 500:
+            assert 'Content-Type' in response_headers
+            if response_headers['Content-Type'] == 'text/xml':
+                parsed_body = ElementTree.fromstring(response_body)
+
+                if SOAPFault.is_soap_fault(parsed_body):
+                    fault: SOAPFault = SOAPFault.from_parsed(response_headers, parsed_body)
+
+                    for error in fault.error_list:
+                        err_text = ', '.join([f'{k}={v}' for k, v in error.items()])
+                        logger.error('0010', f'SOAP Fault returned from Spine: {err_text}')
```

Comment:
Also because you're passing the logger in here there's a possibility that you're going to log two distinct log message types both with the same `{ProcessTag}0010` process keys, I would suggest one of:
- using a very random process key
- allowing the calling workflow to pass the process number into this method,
- or just having this module define it's own logging object. 

I wondering if maybe defining a new logging object is the best, unless there was a specific reason you wanted to pass it in to this method?